components/typography/h/h--alt.hbs

```diff
@@ -0,0 +1,4 @@
+<h3 class="h3--alt">Lorem ipsum</h3>
+<h4 class="h4--alt">Lorem ipsum</h4>
+<h5 class="h5--alt">Lorem ipsum</h5>
+<h6 class="h6--alt">Lorem ipsum</h6>
```

Comment:
In the text of these examples, let's include which header it is for reference. So, like: `Header 3 Lorem Ipsum` or `H3: Lorem Ipsum`, something like that so that it's visually apparent what is what.